### ISSUE TITLE
New SMP DashBoard

### ISSUE THREAD

**Issue by Abhishek-kumar09** [service-mesh-performance/service-mesh-performance#272]
**Current Behavior**

So service Mesh Performance shows rich performance profiles and realtime tests results, but there   is a need of the dashboard for the SMP where results are analysed and charted visually to give better insights to performance results. The current charts are good but are not showing the ample amount of data that the SMP holds power to.

- Design backend for facilitating the dashboard data
- Design frontend for visually appealing dashboard

**Desired Behavior**

Create a new dashboard for Service Mesh Performance. The good context and reference could be SMI Conformance Dashboards

**Comment by DhairyaBahl** [comment]
@leecalcote I would like to work on this issue. Kindly assign it to me. Also kindly guide me a bit on how can I get started this project like building this in my local environment and stuff like that. 

@Abhishek-kumar09 Hello, I am new with Layer5. It would be nice if you can help me out with doubts that I will have while I am trying to understand the codebase.

**Comment by leecalcote** [comment]
@DhairyaBahl I completely missed this message. No one is working on this. Yes, a dashboard using Jekyll is ideal. Are you still interested in progressing here?

**Comment by DhairyaBahl** [comment]
@leecalcote Yes for sure ! I would love to work on this. Kindly give me some resources so that I will quickly get started with this issue.

**Comment by asubedy** [comment]
I would like to work on this issue.  Got some good exposure to Jekyll while contributing on the compatibility matrix.

**Comment by Abhijay007** [comment]
> I would like to work on this issue. Got some good exposure to Jekyll while contributing on the compatibility matrix.

@asubedy I want to collaborate on this :)

**Comment by DhairyaBahl** [comment]
> @DhairyaBahl Please join in Meshery Development call: https://meet.layer5.io/meshery or Community call: https://meet.layer5.io/community and add your question on the meeting minutes to kick start your progress.

I was busy from past one month on my college exams and all and that is why I was unable to give much time to open source.  I will start working on this issue asap.

@asubedy @Abhijay007 are you guys working on this issue ? or should I continue working on this. Kindly inform me.

**Comment by YashKamboj** [comment]
@DhairyaBahl Are you still working on it ??, if yes would you please ping me on slack
